Open Judge System/Web/OJS.Web/Areas/Administration/Controllers/ContestsController.cs

```diff
@@ -675,5 +734,29 @@ private void AddProblemGroupsToContest(Contest contest, int problemGroupsCount)
 
             return message;
         }
+
+        private int GetContestSubmissionsAverageRunTimeSeconds(Contest contest)
+        {
+            List<Submission> contestSubmissions = contest.ProblemGroups
+                                    .SelectMany(pg => pg.Problems)
+                                    .SelectMany(p => p.Submissions)
+                                    .ToList();
```

Comment:
We don't need all the submission's data. Project this to a model with only the properties used for calculating average run time.